frontend/src/components/NavigationBar/NavigationBar.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import '../../styles/NavigationBar/NavigationBar.scss';
+import Accordion from 'react-bootstrap/Accordion';
+import Card from 'react-bootstrap/Card';
+import ListGroup from 'react-bootstrap/ListGroup';
+import Icon from '../../icons/dropdown-arrow.png';
+
+function NavigationBar() {
+    return (
+        <Accordion className="sidebar" style={{ border: '4px' }}>
+            <Card.Header class="card-header" style={{ backgroundColor: 'white' }}>
+                <Card.Link className="card-link" href="/home" style={{ color: '#2F2F31' }}>Home</Card.Link>
```

Comment:
I think Card.Link is just the equivalent of an anchor tag so we're gonna need replace this with something that wraps Link in react router so we can pass around props. We can worry about this later when we have the other pages created - maybe create an issue for this.